changelog.d/11288.bugfix

```diff
@@ -0,0 +1 @@
+Changed the aspect() function to return at least size 1 in width and height.
```

Comment:
Thanks for clarifying! We should edit the changelog to describe the user-visible bugfix. Here's my suggestion.

I've also included your username to give you credit for the fix. Please change if if you'd like to be credited differently, or remove it if you'd prefer not to be credited.

```suggestion
Fix a long-standing bug where uploading extremely thin images (e.g. 1000x1) would fail. Contributed by @Neeeflix.
```